blobstore/src/main/java/org/jclouds/blobstore/options/ListContainerOptions.java

```diff
@@ -58,6 +59,12 @@ public ListContainerOptions() {
       this.delimiter = delimiter;
    }
 
+   ListContainerOptions(Integer maxKeys, String marker, String dir, boolean recursive,
+            boolean detailed, String prefix, String delimiter, boolean versions) {
+      this(maxKeys, marker, dir, recursive, detailed, prefix, delimiter);
+      this.versions = versions;
```

Comment:
Same comment about making this the primary method and add a `@Deprecated` annotation to the old method.